docs/components/Text/Text.tsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
```

Comment:
can you explain why this change was included in this cl? if it's a systematic change it'd be better to make a separate cl for it

Reply:
This update make Euclid our brand font the default font family for this Text component. All components in this repo are used to just helo the documentation and examples of the design tokens that should be the main focus.